### ISSUE TITLE
Regression loss can't decrease

### ISSUE THREAD

**Issue by long-nguyen12** [zylo117/Yet-Another-EfficientDet-Pytorch#662]
Hi, I trained with my custom dataset with one class and I got the result but the reg loss still quite high.
Please. I need ur help!
![image](https://user-images.githubusercontent.com/61386176/122844275-8e563180-d32b-11eb-89d4-090a6ca737e0.png)

**Comment by zylo117** [comment]
have you tested mAP using coco_eval?

**Comment by long-nguyen12** [comment]
> have you tested mAP using coco_eval?

yes, this is my result
![image](https://user-images.githubusercontent.com/61386176/123083401-38c47680-d44a-11eb-8ddf-270a3b689e24.png)

**Comment by zylo117** [comment]
looks good enough...
why would you care about the loss, leave it be

**Comment by long-nguyen12** [comment]
> looks good enough...
> why would you care about the loss, leave it b
